transcoder/transcoder.go

```diff
@@ -12,9 +12,9 @@ import (
 	"strconv"
 	"strings"
 
-	"github.com/xfrr/goffmpeg/ffmpeg"
-	"github.com/xfrr/goffmpeg/models"
-	"github.com/xfrr/goffmpeg/utils"
+	"github.com/Autherz/goffmpeg/ffmpeg"
```

Comment:
Might want to fix these to use xfrr imports?

Reply:
pending untill you fix the imports